launchflow/aws/acm.py

```diff
@@ -32,7 +32,7 @@ class ACMCertificate(AWSResource[ACMCertificateOutputs]):
     ```
     """
 
-    product = ResourceProduct.AWS_ACM_CERTIFICATE
+    product = ResourceProduct.AWS_ACM_CERTIFICATE.value
```

Comment:
optional nit: What if we keep this as an enum on the Resource class, and use the value on just the ResourceState class?

Reply:
Im good leaving as a string too, but reminder to revert the other .value changes throughout the pr if you think we should keep using the enum here

Reply:
I think I'll leave it a string